crates/bdk/src/wallet/mod.rs

```diff
@@ -1175,7 +1175,7 @@ impl Wallet {
     /// let psbt = {
     ///    let mut builder =  wallet.build_tx();
     ///    builder
-    ///        .add_recipient(to_address.script_pubkey(), 50_000);
+    ///        .add_recipient(to_address.script_pubkey(), bitcoin::Amount::from_sat(50_000));
```

Comment:
It's already imported in line 40

```suggestion
    ///        .add_recipient(to_address.script_pubkey(), Amount::from_sat(50_000));
```